float64 point_spacing
geometry_msgs/PoseArray input_poses
---
geometry_msgs/PoseArray output_poses